templates/test/ci/cluster-template-prow-ci-version-windows.yaml

```diff
@@ -435,7 +435,7 @@ spec:
         marketplace:
           offer: capi-windows
           publisher: cncf-upstream
-          sku: k8s-1dot18dot19-windows-2019
+          sku: k8s-1dot19dot5-windows-2019
```

Comment:
where do we see this template was generated?

Reply:
In the file view in the PR

![image](https://user-images.githubusercontent.com/18291632/160453157-30dc35ef-1c50-4d50-96fe-5edda94e615c.png)

Reply:
In previous PRs these files show up as non-generated files (and I usually start adding PR comments to these files instead of the files that are used to generated the templates which isn't helpful in reviews)

Reply:
lgtm